passkeeper/__init__.py

```diff
@@ -77,6 +77,10 @@ def cleanup_ini(self):
                 LOG.info('Clean file %s' % fname)
                 run_cmd('shred --remove %s' % file_path)
 
+    def cleanup_history(self, directory):
+        self.shred_dir(directory)
+        self.git.clean()
```

Comment:
If I understand well, shred remove the "git behavior" of the directory.
After that you call a methode to call
```
git init
git add
git commit
```
I think it's easier to read this instead of a git function doing the job.

I recommend you to call therse 3 command and let git generic without magic function.